examples/internal/proto/examplepb/visibility_rule_none_echo_service.swagger.json

```diff
@@ -112,6 +112,44 @@
           "VisibilityRuleEchoService"
         ]
       }
+    },
+    "/v1/example/echo_hidden": {
+      "get": {
+        "summary": "EchoInternalMessage method is always visible",
+        "operationId": "VisibilityRuleEchoService_EchoInternalMessage",
+        "responses": {
+          "200": {
+            "description": "A successful response.",
+            "schema": {
+              "$ref": "#/definitions/examplepbHiddenMessage"
```

Comment:
This reference is still here, even though the definition is not. Isn't this an invalid OpenAPI spec now?

Reply:
I fixed this